savant/entrypoint/main.py

```diff
@@ -89,3 +89,7 @@ def main(config_file_path: str):
                 os._exit(1)  # pylint: disable=protected-access
     except Exception as exc:  # pylint: disable=broad-except
         logger.error(exc, exc_info=True)
+        exit(1)
```

Comment:
What is the reason for `exit` here? `main` is the only function in entrypoint and there it's your final row

Reply:
`exit` here is needed to stop the module with non-zero exit code.